rosetta-lang/src/main/java/com/regnosys/rosetta/formatting2/CodeFormatterService.java

```diff
@@ -0,0 +1,10 @@
+package com.regnosys.rosetta.formatting2;
+
+import java.io.IOException;
+import java.util.List;
+
+import org.eclipse.emf.ecore.resource.Resource;
+
+public interface CodeFormatterService {
+	List<Resource> formatCollection(List<Resource> resources) throws IOException; 
```

Comment:
If this method modifies the resources in-place, I would change this to `void`. A bit of documentation to explain this behaviour would also be good.